lib/elixir/lib/map.ex

```diff
@@ -15,7 +15,7 @@ defmodule Map do
 
   Maps do not impose any restriction on the key type: anything can be a key in a
   map. As a key-value structure, maps do not allow duplicated keys; keys are
-  compared using the match operator (`===`).
+  compared using the exact equality operator (`===`).
```

Comment:
> Generally, hyphenate two or more words when they come before a noun they modify and act as a single idea. This is called a compound adjective.
> Examples:
> *an off-campus apartment*
> *state-of-the-art design*

(from [random source](http://www.grammarbook.com/punctuation/hyphens.asp) on the interwebs)

I think we should use the `-` here :).